### ISSUE TITLE
Link doesn't do anything

### ISSUE THREAD

**Issue by marcodeluca** [iusvar/grav-plugin-snappygrav#7]
Hi, 

Plugin is installed, absolute urls enabled, and it's not working. I get a link generated like this:

<a title="Get a complete PDF" href="http://fnihb.fractionl.com:completepdf">
<i class="fa fa-file-pdf-o"></i>
</a>

That does nothing in firefox if I click the link, if I try to load the URL directly it takes me to a google search page. In chrome, the link triggers an "about:blank" page.

Any suggestions?

**Comment by iusvar** [comment]
After "I get a link generated like this:" nothing appears.
However, can you tell me what skeleton and theme you are using and the type of printing (:pdf or :completepdf)?
Can you provide link to your website or a screenshot?

**Comment by marcodeluca** [comment]
<a title="Get a complete PDF" href="http://fnihb.fractionl.com:completepdf">
        <i class="fa fa-file-pdf-o"></i>
        PDF
    </a>

http://fnihb.fractionl.com/

the PDF link is in the right sidebar. It's a stock implementation of learn2. Thanks for the quick reply!

**Comment by iusvar** [comment]
Libraries provided in https://github.com/iusvar/grav-plugin-snappygrav#requirements have been installed? This error usually happens when **wkhtmltopdf-i386** is not found

**Comment by marcodeluca** [comment]
Yah both libraries have been installed. I run my own server, ubuntu 14.04, apache, php5, etc. I installed both requirements via composer.

**Comment by iusvar** [comment]
You can check if the `wk_path` variable contains the right address of `wkhtmltopdf-i386` library? The variable is in `snappygrav.yaml` file located in the `user/config/plugins` folder (if you have at least once with the modified Admin), otherwise the original folder `user/plugins/snappygrav`.
It should look something like this: `wk_path: 'vendor/h4cc/wkhtmltopdf-i386/bin/wkhtmltopdf-i386'`

**Comment by pippogsm** [comment]
Hi,

I'm having the same issue:
I'm using GRAV with the lear2 theme, served by XAMPP on a Windows10 machine.
Installed the plugin and the dependencies (I've used this library, since I'm on windows https://github.com/wemersonjanuario/wkhtmltopdf-windows), placed the link in base.html.twig and it appeared in the rendered html as "http://localhost/grav/basics/overview:pdf".
When I click on that link (in google chrome), I get:
```
This site can’t be reached

The webpage at http://localhost/grav/basics/overview:pdf might be temporarily down or it may have moved permanently to a new web address.
ERR_INVALID_RESPONSE
```
And in dev. console:
`GET http://localhost/grav/basics/overview:pdf net::ERR_INVALID_RESPONSE`

Here is my snappygrav.yaml file content:
```
enabled: true           # global enable/disable the entire plugin
wk_path:                'vendor/wemersonjanuario/wkhtmltopdf-windows/bin/32bit'
built_in_css: false     # Use built in CSS
grayscale: false        # PDF will be generated in grayscale if true (default = false)
margin_bottom: 10       # Set the page bottom margin (<unitreal> default 10mm)
margin_left: 10         # Set the page left margin (<unitreal> default 10mm)
margin_right: 10        # Set the page right margin (<unitreal> default 10mm)
margin_top: 10          # Set the page top margin (<unitreal> default 10mm)
orientation: Portrait   # Set orientation to Landscape or Portrait
page_size: A4           # Set paper <size> to: A4, Letter, etc.
title: true             # I prefer True or False, getting title from site page
zoom: 1                 # Use this zoom factor (<float> default 1)
```

I'm suspecting there's something wrong either with the **wk_path** or maybe with the https://github.com/wemersonjanuario/wkhtmltopdf-windows library?

I'm really pressed to get this working asap, so I would greatly appreciate your support.

Thanks.

**Comment by pippogsm** [comment]
Ok, figured it out, it really was the **wk_path** thingy.
I had to set it to: 
`vendor\wemersonjanuario\wkhtmltopdf-windows\bin\64bit\wkhtmltopdf.exe`

So it was was both a problem with forward slash vs backward slash, as well as having to also specify the executable file name.

But now I have an issue with :completepdf, as it seems to taka a really long time and the connection eventually drops. 

The error in console is similar to before:
`http://localhost/grav/basics/requirements:completepdf net::ERR_INVALID_RESPONSE`
While Chrome gives the This site can't be reached message.
Any ideas where to go from here? 

Thanks

**Comment by pippogsm** [comment]
I'm writing this here in case anyone goes trough the same issues as me.

> But now I have an issue with :completepdf, as it seems to taka a really long time and the connection eventually drops.
Fixed this by increasing memory_limit in php.ini to 512M.

Another issue I had was that none of my .md files had the category taxonomy defined, witch resulted in an empty $html object for snappy to convert, so I was getting an error saying $page_slug is not defined.

I fixed it by adding this in the header of my .md files:
```
...
taxonomy:
    category: docs
...
```
